python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorators.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from functools import cached_property
+from typing import Callable, Mapping, Optional
+
+from dagster import AssetsDefinition, multi_asset
+from dagster._core.definitions.events import AssetKey
+
+from .asset_utils import default_asset_key_fn, get_dbt_multi_asset_args, get_deps
+from .utils import ASSET_RESOURCE_TYPES, output_name_fn, select_unique_ids_from_manifest
+
+
+@dataclass
+class DbtManifest:
+    """Helper class for dbt manifest operations."""
+
+    manifest: dict
+
+    @cached_property
+    def dbt_unique_id_to_nodes(self) -> Mapping[str, dict]:
```

Comment:
Naming: we typically name dictionaries with the convention `nodes_by_dbt_unique_id`.

Also, is this an improper use of "node" in dbt nomenclature, given that only a subset of these nodes are `**self.manifest["nodes"]`?

Also also, in the past we've used "node_info" instead of "node". Thoughts on one vs. the other? I think there's value in consistency.